releng/org.eclipse.n4js.releng.utils/src/org/eclipse/n4js/antlr/N4JSAntlrGeneratorFragment2.xtend

```diff
@@ -120,104 +105,4 @@ class N4JSAntlrGeneratorFragment2 extends N4AntlrGeneratorFragment2 {
 		return c2;
 	}
 
-
-	// below is the fix for GH-39 - Content Assist doesn't reliably work in generator functions
-	// TODO will be available with Xtext 2.13 as part of the framework.
-	@Inject ReflectExtensions reflector
-	@Inject extension GrammarAccessExtensions grammarUtil
-	@Inject extension SyntheticTerminalDetector
-
-	private def <T> T reflectiveGet(String fieldName) {
-		try {
-			return reflector.get(this, fieldName);
-		} catch (Exception e) {
-			throw new RuntimeException(e)
-		}
-	}
-
-	protected override boolean hasSyntheticTerminalRule() {
-		grammar.allTerminalRules.exists[isSyntheticTerminalRule]
-	}
-
-	private def ContentAssistGrammarNaming getContentAssistNaming() {
-		return reflectiveGet("contentAssistNaming");
-	}
-
-	private def FileAccessFactory getFileFactory() {
-		return reflectiveGet("fileFactory");
-	}
-
-	private def boolean isPartialParsing() {
-		return <Boolean>reflectiveGet("partialParsing");
-	}
-
-	/**
-	 * This method was copied from the super class to extract the logic for {@link #initNameMappings()}
-	 * which needs access to the flattened grammar.
-	 */
-	override JavaFileAccess generateContentAssistParser() {
-		val extension naming = contentAssistNaming
-		val file = fileFactory.createGeneratedJavaFile(grammar.parserClass)
-		file.content = '''
-			public class «grammar.parserClass.simpleName» extends «grammar.getParserSuperClass(partialParsing)» {
-			
-				«grammar.initNameMappings()»
-			
-				@«Inject»
-				private «grammar.grammarAccess» grammarAccess;
-			
-				@Override
-				protected «grammar.internalParserClass» createParser() {
-					«grammar.internalParserClass» result = new «grammar.internalParserClass»(null);
-					result.setGrammarAccess(grammarAccess);
-					return result;
-				}
-			
-				«IF hasSyntheticTerminalRule»
-					@Override
-					protected «TokenSource» createLexer(«CharStream» stream) {
-						return new «grammar.tokenSourceClass»(super.createLexer(stream));
-					}
-					
-				«ENDIF»
-				@Override
-				protected String getRuleName(«AbstractElement» element) {
-					return nameMappings.getRuleName(element);
-				}
-			
-				@Override
-				protected «Collection»<«"org.eclipse.xtext.ide.editor.contentassist.antlr.FollowElement".typeRef»> getFollowElements(«grammar.internalParserSuperClass» parser) {
-					try {
-						«grammar.internalParserClass» typedParser = («grammar.internalParserClass») parser;
```

Comment:
This was the place where we had a custom code generation.

Reply:
I see, thanks!